glyphs-reader/src/font.rs

```diff
@@ -862,6 +891,51 @@ impl TryFrom<RawGlyph> for Glyph {
     }
 }
 
+impl RawFeature {
+    fn generate_name(&self) -> &'static str {
+        match self.automatic {
+            Some(1i64) => "automatic",
+            _ => "???",
```

Comment:
where did you infer this behavior from?
If created from Glyphs.app, feature prefixes (like classes or features proper) will always have a name in the UI and as such also in the .glyphs source file. That special "<anonymous>" named prefix that you saw in glyphsLib source code is simply the special name that glyphsLib resorts to when converting from features from UFO => .glyphs, for anything that doesn't match the `# Prefix: (.*)$` pattern which is used to record a Glyphs' prefix.name when going .glyphs => UFO.
The automatic bool flag doesn't have anything to do with a RawFeature's name. Also that `"???"` seems arbitrarty to me.
I think you can expect that all RawFeatures (be them prefix, class or features) have a name, it's not optional; then you can get rid of generate_name() method altogether.

Reply:
It stands in for autostr. I suppose complete consistency with that would be "" but I prefer the indicator something is awry. I don't think we _should_ ever hit ???. Maybe I'll add a warn!.